pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[project]
+name = "qoqo_myqlm"
+version = "0.4.3"
+description = "An inteface to execute circuits from QOQO to MYQLM"
+authors = [{name="HQS Quantum Simulation GmbH", email="info@quantumsimulations.de"}]
+readme = "README.md"
+requires-python = ">3.8"
+dependencies = [
+    "qoqo>=1.0.0",
+    "numpy",
+    'myqlm',
+    'qoqo_calculator_pyo3>=1.0.0',
+]
+[project.optional-dependencies]
+tests = [
+  'pytest',
+]
+dev = [
+    'flake8',
+    'flake8-bugbear',
+    'flake8-pyproject',
+    'flake8-pydocstyle',
+    'darglint',
```

Comment:
Check the actions run with this, you may need to add  `    'flake8-bandit',` here